addon/templates/components/polaris-text-style.hbs

```diff
@@ -6,23 +6,23 @@
   component instance once that instance has been created :sadface:
 --}}
 {{#with (component "wrapper-element"
```

Comment:
there's a new `element` helper provided by `ember-element-helper` that should clean this up a bit and get rid of using our custom `wrapper-element` here (check an example [here](https://github.com/smile-io/ember-polaris/pull/360/files#diff-f10a236d2711aa0210af5420b6ff2d76R1))